src/routes.js

```diff
@@ -12,19 +13,33 @@ const AccessRequests = lazy(() => import('./smart-components/accessRequests/acce
 const QuickstartsTest = lazy(() => import('./smart-components/quickstarts/quickstarts-test'));
 
 export const Routes = () => {
+  const location = useLocation();
+  useEffect(() => {
+    insights.chrome.updateDocumentTitle(
+      (Object.values(pathnames).find(
+        (item) =>
+          !!matchPath(location.pathname, {
+            path: item.path,
+            exact: true,
+            strict: false,
+          })
+      )?.title || 'User Access') + ' | console.redhat.com'
+    );
+  }, [location]);
```

Comment:
You should only listen for the `location.pathname` changes. We don't wanna run the effect on each location change (search, hash, etc.)